Orange/widgets/data/owrank.py

```diff
@@ -496,9 +497,11 @@ def on_done(self, result: Results) -> None:
             sort_column, sort_order = self.sorting
             if sort_column < len(labels):
                 # adds 2 to skip the first two columns
-                self.ranksModel.sort(sort_column + 2, sort_order)
+                # in PyQt6 Qt.SortOrder is not int, and so sort_order need to be
+                # transformed to Qt.SortOrder
```

Comment:
Could you also comment that here Qt.SortOrder(sort_order) puts whatever this was to the correct thing? :)

Reply:
Done